src/lib/coda_base/pending_coinbase.ml

```diff
@@ -584,7 +610,7 @@ module T = struct
           (Merkle_tree.add_path t path key Stack.empty)
           (Or_error.ok_exn (Stack_id.incr_by_one key))
     in
-    { tree=
+    { Poly.Stable.Latest.tree=
```

Comment:
I think you can define a `Poly.t`, which the same as `Poly.Stable.Latest.t`, except no `bin_io` or `version`. Then you can just write `Poly.tree` here.